_docker/drupal/drupal-filesystem/web/modules/custom/redhat_developers/src/Twig/RedHatDevelopersTagsExtension.php

```diff
@@ -7,23 +7,25 @@
 
 namespace Drupal\redhat_developers\Twig;
```

Comment:
Can this be tested with PHPUnit or other recommended PHP unit testing framework?